src/components/badges/BadgeBlock.tsx

```diff
@@ -4,9 +4,9 @@ import { useState } from 'preact/hooks'
 import BadgeCard from 'components/badges/BadgeCard'
 import BadgeTitle from 'components/badges/BadgeTitle'
 import BadgeWrapper from 'components/badges/BadgeWrapper'
-import BaseProof from 'helpers/proofs/BaseProof'
-import Button from 'components/Button'
+import Button from 'components/ui/Button'
 import MintedToken from 'models/MintedToken'
+import Proof from 'models/Proof'
```

Comment:
Maybe we have this type in one of our packages?

Reply:
We didn’t put this in a separate package